src/renderers/testing/ReactTestRenderer.js

```diff
@@ -47,6 +49,19 @@ function getRenderedHostOrTextFromComponent(component) {
 }
 
 class ReactTestComponent {
+  _currentElement: ReactElement;
+  _renderedChildren: null | Object;
+  _topLevelWrapper: null | ReactInstance;
+  mountChildren: (
+    children: Object,
+    transaction: ReactTestReconcileTransaction,
+    context: Object,
+  ) => void;
+  updateChildren: (
+    children: Object,
+    transaction: ReactTestReconcileTransaction,
+    context: Object,
+  ) => void;
```

Comment:
These methods are inherited from ReactMultiChild, which isn't typed. Is there a better way tell flow that this class inherits from ReactMultiChild using the current method of calling `Object.assign` on the prototype?